src/Univapay/Resources/TransactionToken.php

```diff
@@ -118,6 +119,17 @@ protected function getIdContext()
         return $this->context->withPath(['stores', $this->storeId, 'tokens', $this->id]);
     }
 
+    protected function pollableStatuses()
+    {
+        return [(string) ThreeDSStatus::PENDING() =>
+            array_diff(ThreeDSStatus::findValues(), [ThreeDSStatus::PENDING()])];
+    }
+
+    protected function statusPropertyPath()
+    {
+        return 'data->threeDS->status';
+    }
```

Comment:
This needs to support both 3DS status and CVV auth status.. how should we go about this?